pkg/Makefile.am

```diff
@@ -41,13 +41,6 @@ $(DIST_STAMP): $(srcdir)/package-lock.json $(PKG_INPUTS)
 
 EXTRA_DIST += build.js files.js package.json package-lock.json
 
-# This is how the qunit tests get included.  We need to prevent automake from
-# seeing them during ./autogen.sh, but need make to find them at compile time.
-# We don't run them in the pybridge case since they're part of `pytest`.
-if !WITH_PYBRIDGE
--include $(wildcard pkg/Makefile.qunit*)
```

Comment:
Can you add a note about the rationale behind this removal in the commit message?

Reply:
Thanks for pointing out. I dropped that while I still thought "distcheck will now run with pybridge", and then realized that we aren't ready for that yet. Put back, and used WITH_OLD_BRIDGE now.